.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Author, Reviewer, and Assignee of a PR
```

Comment:
Can you make this heading a little more descriptive? Something like, "Roles of a PR: Author, Reviewer, and Assignee"